core/chaincode/lifecycle/lifecycle.go

```diff
@@ -305,8 +305,6 @@ func (ef *ExternalFunctions) CommitChaincodeDefinition(chname, ccname string, cd
 		return nil, errors.WithMessage(err, "could not serialize chaincode definition")
 	}
 
-	logger.Infof("Successfully committed definition %s, name '%s' on channel '%s'", cd, ccname, chname)
```

Comment:
Did you remove this because it is printed immediately after the one at line 287 above? If this is the case, isn't this a better message to retain instead (perhaps with a small modification to suit the tone of message at line 448 below)?

Reply:
I removed it from here because the commit may still fail in the caller, based on the returned approvals. In fact, that is what triggered this PR... we saw the commit request failing endorsement, but the peer claimed is was successful in this message. Moving the message until after the approvals have been verified fixes the issue.

Reply:
Ah, I see... this portion of the code split across scc and lifecycle often causes confusion to me.